sky/backends/backend_utils.py

```diff
@@ -2044,6 +2045,27 @@ def get_clusters(
             if record['name'] not in SKY_RESERVED_CLUSTER_NAMES
         ]
 
+    yellow = colorama.Fore.YELLOW
+    bright = colorama.Style.BRIGHT
+    reset = colorama.Style.RESET_ALL
+
+    if cluster_names:
```

Comment:
nit: is not None?

Under it, check if it's a str or a List otherwise throw a TypeError?

Reply:
> nit: is not None?

Fixed. Thanks!

> Under it, check if it's a str or a List otherwise throw a TypeError?

The type argument type should respect the type annotation (will be guaranteed by #1564). Instead of checking the type here, we may need to figure out a way to restrict the input type for each public API (maybe have a decorator that checks the user input arguments following the annotation). How about we add it in a future PR?

Reply:
Sounds good to me.